tools/docker/builder/openwrt/build.sh

```diff
@@ -174,6 +174,7 @@ main() {
 
 }
 
+VERBOSE=true
```

Comment:
This change is not mentioned in the commit message.
I just wanted to mention there was a --verbose option before that set this variable, it has been removed with the shellcheck fixes.
I added the option back in my upcoming PR, but we could also not add the option back and keep this change (as you prefer).

Reply:
That was actually a debugging leftover that I forgot to remove again...